src/components/Cart/CartButtons.jsx

```diff
@@ -0,0 +1,29 @@
+import { useCart } from "../../context";
+import { removeFromCartHandler, updateCartHandler } from "../../utils";
+
+const UpdateCartItem = (props) => {
+    const {cartDispatch} = useCart();
+    
+    const actionTypeIncrement = {
+        action: {type:"increment"}
+    }
+
+    const actionTypeDecrement = {
+        action: {type:"decrement"}
+    }
+    return (
+        <>
+        {
+            props.btnType==="delete" &&  <i className="fa-solid fa-trash" onClick={(element) => removeFromCartHandler(element, props.productId, props.token, cartDispatch )}></i>
+        }
+        {
+            props.btnType==="decrement" &&  <i className="far fa-minus-square" onClick={(element) => updateCartHandler(element, props.productId, props.token, cartDispatch, actionTypeDecrement)}></i>
+        }
+        {
+            props.btnType==="increment" &&  <i className="far fa-plus-square" onClick={(element) => updateCartHandler(element, props.productId, props.token, cartDispatch, actionTypeIncrement)}></i>
+        }
+        </>
+    )
+}
+
```

Comment:
This is a very good way of removing buttons from the Cart component and adding it as a separate component! Makes your code cleaner.